scootapi/client/run_job_cmd.go

```diff
@@ -33,13 +33,15 @@ func (c *runJobCmd) registerFlags() *cobra.Command {
 
 // Types to handle JobDefinitions from JSON files
 type JobDef struct {
-	Tasks map[string]TaskDef
+	Tasks     map[string]TaskDef
+	TimeoutMs int32
```

Comment:
Ah gotcha - these get resolved in scootapi/server/run_job.go - task timeout will be used, if it's not set then job timeout will be used